app/views/unattended/provisioning_templates/snippet/ca_registration.erb

```diff
@@ -26,6 +26,9 @@ description: |
     cp $KATELLO_SERVER_CA_CERT $CA_TRUST_ANCHORS
 
     if [ -f /etc/debian_version ]; then
+      # update-ca-certificates is picky and only looks at .crt files
```

Comment:
Wouldn't it be better/easier, to change the original file name in line 13 like
```
if [ -f /etc/debian_version ]; then
  KATELLO_SERVER_CA_CERT=/etc/rhsm/ca/katello-server-ca.crt
else
  KATELLO_SERVER_CA_CERT=/etc/rhsm/ca/katello-server-ca.pem
fi
```

Reply:
Not quiet, because we then also would have to change [subscription_manager_setup.erb](https://github.com/theforeman/foreman/blob/develop/app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb#L52), where the file is 'touched'.